source/jobs/worker_or_atools.rst

```diff
@@ -0,0 +1,176 @@
+
++------------+--------+--------+
+| walltime   | worker | atools |
++============+========+========+
+| < 1 second | \-     | \-\-   |
++------------+--------+--------+
+| < 1 minute | \+     | \-     |
++------------+--------+--------+
+|            | \+\+   | \+\+   |
++------------+--------+--------+
+| > 24 hours | \+     | \+\+   |
++------------+--------+--------+
+
+
+Number of work items
+~~~~~~~~~~~~~~~~~~~~
+
+If you need to do many individual computations (work items), say more than
+500, worker is the better choice.  It will be run as a single job, rather
+than many individual jobs, hence lightening the load on the scheduler
+considerably.
+
+
+Job policies
+~~~~~~~~~~~~
+
+The following job policies are currently in effect on various VSC clusters.
+
+Shared
+   Multiple jobs from multiple users are allowed to run concurrently
+   on a node.
+Single user
+   Multiple jobs from a single user are allowed to run concurrently
+   on a node.
+Single job
+   Only a single job can run on a compute node.
+
+On some clusters, credits are required to run jobs, and that policy may
+also influence your choice.
+
+The table below provides an overview of the policies in effect on the
+various cluster/partitions.
+
++----------+--------------------+-----------+-------------+------------+
+| VSC hub  | cluster            | partition | policy      | accounting |
++==========+====================+===========+=============+============+
+| Antwerp  | any                | any       | single user | no         |
++----------+--------------------+-----------+-------------+------------+
+| Brussels | any                | any       | shared      | no         |
++----------+--------------------+-----------+-------------+------------+
+| Ghent    | any                | any       | shared      | no         |
++----------+--------------------+-----------+-------------+------------+
+| Leuven   | thinking           | default   | single user | yes        |
++----------+--------------------+-----------+-------------+------------+
+| Leuven   | genius             | default   | single user | yes        |
++----------+--------------------+-----------+-------------+------------+
+| Leuven   | genius             | bigmem    | single user | yes        |
++----------+--------------------+-----------+-------------+------------+
+| Leuven   | genius             | gpu       | shared      | yes        |
++----------+--------------------+-----------+-------------+------------+
+| Leuven   | genius             | superdome | shared      | yes        |
++----------+--------------------+-----------+-------------+------------+
+| Leuven   | breniac (Tier-1)   | default   | single job  | yes        |
++----------+--------------------+-----------+-------------+------------+
+
+Clusters with accounting enabled:
+   If you use atools on a cluster where accounting is active, make sure a work
+   item uses all resources of that node.  If multiple work items run on the same
```

Comment:
I guess you mean 'all cores of that node'?

Reply:
No, I don't.  A work item may require more memory than RAM/cores.  Hence all memory would be used, but not all cores.